### ISSUE TITLE
samples_weak and samples_strong

### ISSUE THREAD

**Issue by yichenwang231** [ManiadisG/DivClust#1]
https://github.com/ManiadisG/DivClust/blob/b36203a90a9753674a447cfe5edfdd9297314a58/engine/trainer.py#L32
the samles size is [256, 2, 3, 224, 224],samples_weak and samples_strong size is [2, 3, 224, 224] after 
samples_weak = samples[0].to(device,non_blocking=True)
# samples_strong = torch.cat(samples[1:],dim=0).to(device,non_blocking=True)
and this code https://github.com/ManiadisG/DivClust/blob/b36203a90a9753674a447cfe5edfdd9297314a58/engine/trainer.py#L33C1-L33C1 maybe incorrect.

**Comment by ManiadisG** [comment]
Hi @yichenwang231.

Thank you for bringing this to my attention! That line was indeed incorrect, I missed it when I added PICA. It is now fixed and a bit cleaner about how PICA and CC operate differently.
